_docs/geyser/geyser-api.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Geyser API
+---
+
+### Geyser has an API to extend what is possible and allow to listen to Geyser events. It also allows access to [Cumulus](/floodgate/forms/)
```

Comment:
This should not be a subheading. Would break into `Events` and `Forms` with relevant info linked below each subheading for each.